### PR TITLE
Add support for Log.verbose.

### PR DESCRIPTION
**Pull request by shaper**
If there is a reason we haven't included this let's discuss. I thought having the additional level would be helpful as I've spent more time amidst our logging.

### CODE REVIEW

File: java/arcs/android/util/AndroidLog.kt [hunk 1]
```diff
@@ -13,22 +13,25 @@ package arcs.android.util
 
 import arcs.core.util.Log
 
+private val TAG = "Arcs"
+
 /** Initializes [Log] for tests on the JVM. */
-fun initLogForAndroid(level: Log.Level = mapAndroidLogLevel("Arcs")) {
+fun initLogForAndroid(level: Log.Level = mapAndroidLogLevel(TAG)) {
```
Comment (jasonwyatt) [inline java/arcs/android/util/AndroidLog.kt:19]: I think we should still retain at least an `Arcs` prefix on the logcat tag.

Comment (shaper) [reply java/arcs/android/util/AndroidLog.kt:19]: Even with this change we do, don't we? My intent was just to move it to a constant so we can have it in one place for here and in the mapping below.

File: java/arcs/android/util/AndroidLog.kt [hunk 2]
```diff
@@ -13,22 +13,25 @@ package arcs.android.util
...
     Log.level = level
     Log.writer = { lvl, renderedMessage, throwable ->
         when (lvl) {
-            Log.Level.Debug -> android.util.Log.d("Arcs", renderedMessage, throwable)
-            Log.Level.Info -> android.util.Log.i("Arcs", renderedMessage, throwable)
-            Log.Level.Warning -> android.util.Log.w("Arcs", renderedMessage, throwable)
-            Log.Level.Error -> android.util.Log.e("Arcs", renderedMessage, throwable)
-            Log.Level.Wtf -> android.util.Log.wtf("Arcs", renderedMessage, throwable)
+            Log.Level.Debug -> android.util.Log.d(TAG, renderedMessage, throwable)
```
Comment (ianull) [inline java/arcs/android/util/AndroidLog.kt:23]: Add this for the new verbose level?
Log.Level.Verbose -> android.util.Log.v(TAG, renderedMessage, throwable)

Comment (shaper) [reply java/arcs/android/util/AndroidLog.kt:23]: Ah, thanks for catching! I went straight into a meeting and missed adding this line before sending.

**Comment by shaper** [comment]
gcbrun

**Pull request merged**